JavaScript/Markdown_To_HTML/script.js

```diff
@@ -0,0 +1,19 @@
+const marked = require("marked")
+const fs = require("fs")
+
+console.log("Reading from file markdown.md...")
+fs.readFile("markdown.md", "utf-8", (err, data) => {
+    if(err) {
+        console.error(err)
+        return
+    }
+    console.log("Writing into file html.html...")
+    fs.writeFile("html.html", marked(data, { sanitiser: true }), err => {
```

Comment:
Can you change the name to `index.html` or common? html.html can be a bit confusing.

Reply:
I've renamed it to index.html